src/chkstat.c

```diff
@@ -202,6 +202,11 @@ add_level(char *e)
 {
   if (in_level(e))
     return;
+  if (strcmp(e, "paranoid") == 0)
+    {
+      e = "restrictive";
+      fprintf(stderr, "profile 'paranoid' is configured, transparently redirecting this to the new 'restrictive' profile.\nPlease adjust /etc/sysconfig/security to avoid this warning.\n");
```

Comment:
Personally, I'd remove this message. It'd come up during a lot of package updates, no?

Reply:
Only if somebody actually uses the paranoid profile which is probably a rare occasion at the moment. I could only print it if stdin is a terminal or something, or if chkstat runs in `--system` mode.

I just didn't want to completely silently redirect the profile name which could be quite surprising in a couple of years from now ;).

Reply:
After some more discussion a safer approach would be to print a warning on rpm level when the paranoid profile is still configured. This shows a single message during updating.